src/main/java/seedu/address/model/entry/Entry.java

```diff
@@ -103,20 +103,20 @@ public boolean equals(Object other) {
                 && otherEntry.getTag().equals(getTag());
     }
 
-    @Override
-    public String toString() {
-        final StringBuilder builder = new StringBuilder();
-        builder.append(getDescription())
-                .append("; Date: ")
-                .append(getDate())
-                .append("; Amount: ")
-                .append(getAmount());
-
-        Tag tag = getTag();
-        if (tag != null) {
-            builder.append("; Tag: ");
-            builder.append(tag);
-        }
-        return builder.toString();
-    }
+    //    @Override
+    //    public String toString() {
+    //        final StringBuilder builder = new StringBuilder();
+    //        builder.append(getDescription())
+    //                .append("; Date: ")
+    //                .append(getDate())
+    //                .append("; Amount: ")
+    //                .append(getAmount());
+    //
+    //        Tag tag = getTag();
+    //        if (tag != null) {
+    //            builder.append("; Tag: ");
+    //            builder.append(tag);
+    //        }
+    //        return builder.toString();
+    //    }
```

Comment:
Is this `toString` method still needed, since our entries are abstract, we would (likely) not need this?

Reply:
yup I don't think we need this anymore